File: 0643-maximum-average-subarray-i/solution.java
class Solution {
    public double findMaxAverage(int[] arr, int k) {
        double max=-1e9;
        for(int i=0;i<=arr.length-k;i++){
            int sum=0;
            for(int j=i;j<i+k;j++){
                sum=sum+arr[j];
            }
           double avg=(double)sum/k;
            max=Math.max(max,avg);
        }
        return (double)max;
    }
}

File: 0643-maximum-average-subarray-i/solution.py
class Solution:
    def findMaxAverage(self, nums: List[int], k: int) -> float:
        w_sum=(sum(nums[:k]))
        m_sum=w_sum/k
        for i in range(k,len(nums)):
            w_sum=w_sum+nums[i]
            w_sum=w_sum-nums[i-k]
            m_sum=max(m_sum,(w_sum/k))
        return m_sum
